assets/admin/editor-wizard/helpers.js

```diff
@@ -188,3 +189,30 @@ export const useHideEditorWizardUpsell = () => {
 		! senseiProExtension || senseiProExtension.is_activated === true
 	);
 };
+
+/**
+ * Hook to get the site slug.
+ *
+ * @return {string} Site slug.
+ */
+export const useSiteSlug = () => {
```

Comment:
Not sure if `SiteSlug` makes sense for other scenarios out of WPCOM. Maybe to make it more clear, could we call it as `useWpcomSiteSlug`?

We could also use only this hook in the components, returning `null` if it's not a WPCOM site. WDYT? Just to simplify since we'll need to have this logic of slug + wpcom check in many places.